sql/branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+    UPDATE payment_instructions SET due = floor(9.81/(amount)) * amount WHERE due > 9.81;
```

Comment:
https://github.com/gratipay/gratipay.com/blob/master/gratipay/billing/exchanges.py#L20

Oh, 9.41. I'll make the change soon